src/ui/svg/icons/CreateAccount.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import Svg, { Path, G } from 'react-native-svg'
+
+import { ColorsEnum } from 'ui/theme'
+
+import { IconInterface } from './types'
+
+export const CreateAccount: React.FunctionComponent<IconInterface> = ({
+  size = 127,
+  color = ColorsEnum.BLACK,
+  testID,
+}) => (
+  <Svg
+    width={Math.round(1.55 * (size as number))}
```

Comment:
1. Il sort d'ou ce calcul ?
2. 
```suggestion
    width={Math.round(1.55 * size))}
```

Reply:
C'est un ratio, sans le cast il y a un warning

Reply:
Le calcul du ratio c'est `viewBox.width / viewBox.height`